src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java

```diff
@@ -84,6 +84,13 @@
         = "EndpointMessage";
 
     /**
+     * The {@link Videobridge#COLIBRI_CLASS} value indicating a
+     * {@code ReceiverVideoConstraint} message.
+     */
+    public static final String RECEIVER_VIDEO_CONSTRAINT
```

Comment:
I know it's technically unnecessary, but can we rename this to something like `COLIBRI_CLASS_VIDEO_CONSTRAINT`, since all the other messages have that prefix?